/**:
  ros__parameters:
    raw_odometry:
      use_gps: false  # Use GPS data
      set_origin_on_start: false  # Set origin of the map frame with the first GPS data
      # set_origin: # Set manually origin of the map frame
      #   lat: 0.0  # Set manually origin latitude
      #   lon: 0.0  # Set manually origin latitude
      #   alt: 0.0  # Set manually origin altitude in meters
      # earth_to_map_height: 0.0  # Set manually height of the map frame above the earth frame
      earth_to_map:
        x: 0.0 # Set manually x offset of the map frame from the earth frame
        y: 0.0 # Set manually y offset of the map frame from the earth frame
        z: 0.0 # Set manually z offset of the map frame from the earth frame

